src/Mod/Points/App/AppPointsPy.cpp

```diff
@@ -414,8 +413,7 @@ class Module: public Py::ExtensionModule<Module>
                 pcDoc = App::GetApplication().newDocument();
             }
             PointsPy* pPoints = static_cast<PointsPy*>(pcObj);
-            Points::Feature* pcFeature =
-                static_cast<Points::Feature*>(pcDoc->addObject("Points::Feature", name));
+            Points::Feature* pcFeature = pcDoc->addObject<Points::Feature>(name);

```

Comment:
Few lines above `auto*` was used, here not - would be great to be consistent

Reply:
I do agree that consistency is important, but in this PR I had to find a balance between use of auto and making PR as small as possible to make it reviewable.

I decided that it was better to avoid converting code to use auto in the regex commit. Doing such a change would break a lot of consistency and if I manually fixes stuff, then it wouldn't be a regex commit anymore.

---

I changed this one and moved it out of the regex commit